vault/dynamic_system_view.go

```diff
@@ -219,7 +218,7 @@ func (d dynamicSystemView) LookupPlugin(ctx context.Context, name string, plugin
 		return nil, err
 	}
 	if r == nil {
-		return nil, errwrap.Wrapf(fmt.Sprintf("{{err}}: %s", name), ErrPluginNotFound)
+		return nil, fmt.Errorf("%s: %w", name, ErrPluginNotFound)
```

Comment:
While I understand that the format used in this error is unconventional, I would prefer that this PR be a drop-in replacement with only s/wrapf/errorf/.  Makes it easier to have confidence in it despite it being such a sweeping change.

Reply:
I understand that. I've made the change.